language/German/strings.po

```diff
@@ -1787,102 +1821,102 @@ msgstr "Weitere Bedienelemente"
 #: src/frontend/mame/ui/inputmap.cpp:58
 #, c-format
 msgid "Input Assignments (%1$s)"
-msgstr ""
+msgstr "Eingabezuweisungen (%1$s)"
 
 #: src/frontend/mame/ui/inputmap.cpp:152
 msgid "Input Assignments (this system)"
-msgstr ""
+msgstr "Eingabezuweisungen (dieses System)"
 
 #: src/frontend/mame/ui/inputmap.cpp:254
 msgid "[no assignable inputs are enabled]"
-msgstr ""
+msgstr "[keine zuweisbaren Eingaben aktiviert]"
 
 #: src/frontend/mame/ui/inputmap.cpp:348
 msgid "Pressed"
-msgstr ""
+msgstr "Gedrückt"
 
 #: src/frontend/mame/ui/inputmap.cpp:418
 msgid "Invalid combination entered"
-msgstr ""
+msgstr "Ungültige Kombination eingegeben"
 
 #: src/frontend/mame/ui/inputmap.cpp:563
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog"
-msgstr ""
+msgstr "%1$s Analog"
 
 #: src/frontend/mame/ui/inputmap.cpp:564
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog Inc"
-msgstr ""
+msgstr "%1$s Analog Inkrement"
```

Comment:
"Erhöhen (vergrößern)/Verringern (reduzieren)" instead of "inkrement/dekrement"?

Reply:
Yeah wasn't sure if I really want Erhöhen / Verringern, but am fine with them.
Fixed.